src/lsp/cobol_lsp/lsp_request.ml

```diff
@@ -677,6 +677,35 @@ let handle_codelens registry ({ textDocument; _ }: CodeLensParams.t) =
     end
   |> Option.value ~default:[]
 
+(** { Rename } *)
+
+exception CopybookRenameError
```

Comment:
Since it's internal to `handle_rename` I think you can even `let exception CopybookRenameError in …`.

Reply:
But for such a local use with so few lines in the function we typically use `Exit`.